src/common/signing.props

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildType Condition="Exists('..\..\script\src\MetricsService.cs')">Internal</BuildType>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(BuildType)' == 'Internal'">
+    <AssemblyOriginatorKeyFile>..\..\script\Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(BuildType)' != 'Internal' and Exists('..\..\publickey.snk')">
```

Comment:
That means changing our CI to generate a key before end for public builds so that we can actually build external PRs, otherwise builds will fail. I've added a note on the README saying that the public key is required, and this has been broken since we disabled the assembly resolver, so I think external builds can survive another week or two until we get our release in shape, and then we can add the checks we need to make sure all our builds are correct.